docs/Shipment.java

```diff
@@ -0,0 +1,53 @@
+public class Shipment(){
+	
+	String shipmentId; 
+	String shipmentMethod;	
+	String currentWarehouseId;
+	float shipmentWeight;
+	long receivedAt;
```

Comment:
Let's declare these private in the class.

Reply:
^agreed on setting these as private